anxcloud/testutils/environment/prefix.go

```diff
@@ -0,0 +1,93 @@
+package environment
+
+import (
+	"context"
+	"fmt"
+	"go.anx.io/go-anxcloud/pkg/client"
+	"go.anx.io/go-anxcloud/pkg/ipam/prefix"
+	"net"
+	"sync"
+	"time"
+)
+
+type Prefix struct {
+	ID      string
+	CIDR    string
```

Comment:
why not store the parsed CIDR instead of parsing it over and over again, having to handle errors way more often?

Reply:
ack